packages/smart-wallet/src/marshal-contexts.js

```diff
@@ -364,7 +364,7 @@ export const makeImportContext = (makePresence = defaultMakePresence) => {
 const makePresence = (iface, handler) => {
   let obj;
   // eslint-disable-next-line no-new
-  new HandledPromise((resolve, reject, resolveWithPresence) => {
+  void new HandledPromise((resolve, reject, resolveWithPresence) => {
```

Comment:
here's what changed since your review @dckc 

I'm going to merge with the prior approval. I'll follow up if you think this should be different.